### ISSUE TITLE
[pre-release 1.1.0] test_augment failure on windows

### ISSUE THREAD

**Issue by IAlibay** [MDAnalysis/mdanalysis#3248]
Testing the 1.1.0 pre-release on windows we have the following issue:

_________________________________________________________________________________________________________ test_augment[q5-res5-b0] _________________________________________________________________________________________________________
b = array([10., 10., 10., 90., 90., 90.], dtype=float32), q = array([[1., 1., 1.]], dtype=float32), res = [[1.1, 0.1, 0.1], [0.1, 1.1, 0.1], [0.1, 0.1, 1.1], [0.1, 1.1, 1.1], [1.1, 1.1, 0.1], [1.1, 0.1, 1.1], ...]
    @pytest.mark.parametrize('b', (
                             np.array([10, 10, 10, 90, 90, 90], dtype=np.float32),
                             np.array([10, 10, 10, 45, 60, 90], dtype=np.float32)
                             ))
    @pytest.mark.parametrize('q, res', qres)
    def test_augment(b, q, res):
        radius = 1.5
        q = transform_StoR(np.array(q, dtype=np.float32), b)
        if q.shape == (3, ):
            q = q.reshape((1, 3))
        q = apply_PBC(q, b)
        aug, mapping = augment_coordinates(q, b, radius)
        if aug.size > 0:
            aug = np.sort(aug, axis=0)
        else:
            aug = list()
        if len(res) > 0:
            cs = transform_StoR(np.array(res, dtype=np.float32), b)
            cs = np.sort(cs, axis=0)
        else:
            cs = list()
>       assert_almost_equal(aug, cs, decimal=5)
E       AssertionError:
E       Arrays are not almost equal to 5 decimals
E
E       Mismatched elements: 4 / 21 (19%)
E       Max absolute difference: 10.
E       Max relative difference: 10.
E        x: array([[ 1.,  1., -9.],
E              [ 1.,  1., -9.],
E              [ 1.,  1.,  1.],...
E        y: array([[ 1.,  1.,  1.],
E              [ 1.,  1.,  1.],
E              [ 1.,  1.,  1.],...
anaconda3\envs\test-mda-3.8\lib\site-packages\MDAnalysisTests\lib\test_augment.py:92: AssertionError
_________________________________________________________________________________________________________ test_augment[q5-res5-b1] _________________________________________________________________________________________________________
b = array([10., 10., 10., 45., 60., 90.], dtype=float32), q = array([[1.5      , 1.7071068, 0.5      ]], dtype=float32), res = [[1.1, 0.1, 0.1], [0.1, 1.1, 0.1], [0.1, 0.1, 1.1], [0.1, 1.1, 1.1], [1.1, 1.1, 0.1], [1.1, 0.1, 1.1], ...]
    @pytest.mark.parametrize('b', (
                             np.array([10, 10, 10, 90, 90, 90], dtype=np.float32),
                             np.array([10, 10, 10, 45, 60, 90], dtype=np.float32)
                             ))
    @pytest.mark.parametrize('q, res', qres)
    def test_augment(b, q, res):
        radius = 1.5
        q = transform_StoR(np.array(q, dtype=np.float32), b)
        if q.shape == (3, ):
            q = q.reshape((1, 3))
        q = apply_PBC(q, b)
        aug, mapping = augment_coordinates(q, b, radius)
        if aug.size > 0:
            aug = np.sort(aug, axis=0)
        else:
            aug = list()
        if len(res) > 0:
            cs = transform_StoR(np.array(res, dtype=np.float32), b)
            cs = np.sort(cs, axis=0)
        else:
            cs = list()
>       assert_almost_equal(aug, cs, decimal=5)
E       AssertionError:
E       Arrays are not almost equal to 5 decimals
E
E       Mismatched elements: 11 / 21 (52.4%)
E       Max absolute difference: 14.142136
E       Max relative difference: 10.
E        x: array([[-3.5    , -5.36396, -4.5    ],
E              [ 1.5    , -5.36396, -4.5    ],
E              [ 6.5    ,  1.70711,  0.5    ],...
E        y: array([[ 1.5    ,  1.70711,  0.5    ],
E              [ 6.5    ,  8.77817,  0.5    ],
E              [ 6.5    ,  8.77817,  0.5    ],...
anaconda3\envs\test-mda-3.8\lib\site-packages\MDAnalysisTests\lib\test_augment.py:92: AssertionError

python version: 3.8
compiler: Visual Studio 2019

**Comment by IAlibay** [comment]
Adding a 2.0 milestone here. As discussed with @richardjgowers we don't think this leads to significant changes in behaviour, just that things get wrapped on a different side of the box.

I'm not sure how we're going to test this one tbh, as far as I am aware my Windows machine is the only thing that captures this. @tylerjereddy are there VM options for MSC 19+ for azure pipelines?

**Comment by IAlibay** [comment]
@jbarnoud might be good to see if you can even reproduce this. It could be a niche toolchain specific issue (I don't usually use Windows itself to run MDAnalysis [being mostly a WSL person]).

**Comment by IAlibay** [comment]
We're going to have to move the goalpost on this one since I can't seem to properly reproduce the error. - setting to 2.1.0

**Comment by IAlibay** [comment]
Note still an issue on 2.0.0.

**Comment by tylerjereddy** [comment]
> @tylerjereddy are there VM options for MSC 19+ for azure pipelines?

Yes, just seeing this now sorry--let me see if I can capture that error in a CI matrix entry now

**Comment by hmacdope** [comment]
See #3361 for possible loss of precision in `transform_StoR()`
